src/tests/test_config.py

```diff
@@ -34,3 +34,9 @@ def test_safeGetInt(self):
             BMConfigParser().safeGetInt('nonexistent', 'nonexistent'), 0)
         self.assertEqual(
             BMConfigParser().safeGetInt('nonexistent', 'nonexistent', 42), 42)
+
+    def test_reset(self):
+        """safeGetInt retuns provided default for bitmessagesettings option or 0"""
+        self.config = BMConfigParser()
```

Comment:
maybe just
`config = BMConfigParser()`

Reply:
And then
```python
config.safeGetInt('bitmessagesettings', 'settingsversion')
```
in the following line.

I suggested `self.config = BMConfigParser()` in `.setUp()` if you want to reuse it in the rest of methods. Here it is redundant.